metadata-ingestion/src/datahub/metadata/schema.avsc

```diff
@@ -5219,20 +5219,20 @@
             "fields": [
               {
                 "type": [
-                  "null",
-                  "long"
+                  "long",
```

Comment:
Where are all these changes coming from?

Reply:
what the...

Reply:
these recently got in when I merged with master. something is off.

Reply:
https://github.com/linkedin/datahub/pull/3035#issuecomment-893320703